refiner-app/src/config.rs

```diff
@@ -46,3 +48,51 @@ pub enum Network {
     Mainnet,
     Testnet,
 }
+
+#[derive(Clone, Debug)]
+pub struct ValidatedAccountId(AccountId);
```

Comment:
Sorry. I can't get why you need this wrapper. Why don't use just `AccountId` here?